telemetry/definitions/commonDefinitions.json

```diff
@@ -982,6 +982,11 @@
             "type": "string",
             "description": "The API request ID"
         },
+        {
```

Comment:
Blocker: We can probably just re-use an existing field like `codeTransformRunTimeLatency` or a more generic one like `duration`. Right now all of our events use the `codeTransformRunTimeLatency`.

I would suggest against new types, unless necessary because we have to modify backend to receive new "types". But not "metrics".

Reply:
Changed to reuse "codeTransformRunTimeLatency".